core-bundle/src/Twig/Slots/SlotTokenParser.php

```diff
@@ -40,8 +41,16 @@ public function parse(Token $token): Node
 
         $body = $this->parser->subparse($this->decideForFork(...));
 
+        $throwMissingSlotFunctionError = static function () use ($nameToken, $token, $stream): void {
+            throw new SyntaxError(\sprintf('Slot "%s" defines template content but is missing a call to the "slot()" function.', $nameToken->getValue()), $token->getLine(), $stream->getSourceContext());
```

Comment:
```suggestion
            throw new SyntaxError(\sprintf('Slot "%s" adds content but does not call the "slot()" function.', $nameToken->getValue()), $token->getLine(), $stream->getSourceContext());
```

Reply:
Can we leave the `template` bit in? What I want to express is, that there was template content (as in "content with a placeholder") added/defined inside the slot tag and it misses the function - contrary to having a slot like `{% slot foo %}{% endslot %}` where it is OK to leave the function out. Both versions could have content assigned in the template parameters (i.e. the thing that gets output in the slot function or the slot body) and I don't want people to confuse these. 

```suggestion
            throw new SyntaxError(\sprintf('Slot "%s" adds template content but does not call the "slot()" function.', $nameToken->getValue()), $token->getLine(), $stream->getSourceContext());
```